src/components/shared/VirtualList.js

```diff
@@ -375,6 +375,20 @@ export class VirtualList<Item> extends React.PureComponent<
     }
   }
 
+  /* This method is used by users of this component. */
+  /* eslint-disable-next-line react/no-unused-class-component-methods */
+  scrollToItem(itemIndex: number, offsetX: CssPixels) {
```

Comment:
What's the difference with the existing `scrollItemIntoView`? It looks like a duplicate :-)

Reply:
`scrollItemIntoView` doesn't necessarily put the item at the top of the viewport. It scrolls just enough that the item becomes visible. So the item is currently below the bottom of the viewport, it'll scroll so that the item is visible at the bottom. But I needed a scrolling function which "resets" the state, so it needs to ignore the old scroll position.

Reply:
I'll add a comment.

Reply:
Added two comments, in a new commit.